src/main/resources/db/migration/v0.34.0.4__rename_orthology_to_homology_vocabulary.sql

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I think you need to create VocabularyTermSets here for OrthologyPredictionMethod and ParalogyPredictionMethod.  The reason being that, based on your previous migration and the current orthology prediction method terms, SGD is a valid term for paralogy but not for orthology.  Conversely Hieranoid, Xenbase, and ZFIN are only valid terms for orthology.

Reply:
Yes. I missed out on this. I will create separate predictionmethods for orthology and paralogy. But, is it vocabularyterm or vocabularytermsets table that needs contain this data?

Reply:
The Vocabulary contains all the terms.  You then have two VocabularyTermSets that have member terms - these are the overlapping subsets of terms in that vocabulary that apply to orthology/paralogy